x-pack/plugins/lens/public/embeddable/expression_wrapper.tsx

```diff
@@ -158,3 +169,52 @@ export function ExpressionWrapper({
     </I18nProvider>
   );
 }
+
+const SavedObjectConflictMessage = ({ json }: { json: string }) => {
+  const [expandError, setExpandError] = useState(false);
+  return (
+    <>
+      <FormattedMessage
+        id="xpack.lens.embeddable.legacyURLConflict.longMessage"
+        defaultMessage="You have encountered a legacy URL alias conflict. See the {documentationLink} for more information."
```

Comment:
Perhaps we'll want to get the @elastic/kibana-docs team's input on this?

(see PR description above for screenshot!)